### PR TITLE
feat(icons): add keyboard-off

### PR DESCRIPTION
**Pull request by Diottodev**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other:

### Description
The opposite of `keyboard` icon

### Icon use case 
The purpose of this icon is to represent closing/disabling/hiding virtual keyboards

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [x] I've based them on the following Lucide icons: keyboard
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to two points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Doesn't follow the style of other `-off` icons, there should be a 2px gap of empty space right of the line.

**Comment by dan1xk** [comment]
this is very useful!

File: icons/keyboard-off.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Diottodev",
+    "karsa-mistmere"
+  ],
+  "tags": [
+    "unkeys"
```
Comment (ericfennis) [inline icons/keyboard-off.json:8]: This should include at least all the tags from the `keyboard` icon. And maybe we can think of some more.

File: icons/keyboard-off.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  ],
+  "categories": [
+    "devices"
```
Comment (ericfennis) [inline icons/keyboard-off.json:11]: Same here, the categories should be based on categories from the `keyboard`  icon.

**Review by ericfennis — CHANGES_REQUESTED** [review]
Nice icon!!
The metadata should be updated

**Review by ericfennis — APPROVED** [review]

**Review by jguddas — APPROVED** [review]

**Pull request merged**